packages/itmat-interface/src/graphql/core/userCore.ts

```diff
@@ -17,8 +17,8 @@ export class UserCore {
         return user;
     }
 
-    public async createUser(requester: string, user: { password: string, username: string, organisation: string, type: userTypes, description: string, realName: string, email: string, emailNotificationsActivated: boolean }): Promise<IUserWithoutToken> {
-        const { password, organisation, username, type, description, realName, email, emailNotificationsActivated } = user;
+    public async createUser(requester: string, user: { password: string, username: string, organisation: string, type: userTypes, description: string, realName: string, email: string, emailNotificationsActivated: boolean, createdAt: number, expiredAt: number, locked: boolean }): Promise<IUserWithoutToken> {
```

Comment:
It in unclear why we would need to pass in `createdAt` in the function that will create the user. Isn't that something the function should derive on its own ?

Reply:
Sure, would correct that.